packages/atlas/src/components/_video/VideoTilePublisher/VideoTilePublisher.tsx

```diff
@@ -304,6 +308,12 @@ export const VideoTilePublisher: FC<VideoTilePublisherProps> = memo(
       return videoHref
     }, [hasVideoUploadFailed, isSyncingWithYoutube, isUploading, videoHref])
 
+    useEffect(() => {
+      if (uploadVideoStatus?.lastStatus === 'completed' || uploadThumbnailStatus?.lastStatus === 'completed') {
+        refetchVideo()
+      }
+    })
```

Comment:
maybe should add those 2 to deps so it wouldn't trigger every time, even though it would return after the first line

Reply:
Definitely